components/teleport/NetworkDropdown.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div class="w-full">
```

Comment:
```suggestion
  <div>
```
I don't think it's needed